src/fn/compile-app-settings.js

```diff
@@ -12,8 +12,10 @@ const compileAppSettings = async (projectDir, couchUrl, extraArgs) => {
   const options = parseExtraArgs(extraArgs);
   projectDir = path.resolve(projectDir);
   
-  checkMedicConfDependencyVersion(projectDir);
-
+  if(options.dependencyCheck){
```

Comment:
Is compile-app-settings the right place to do this?

Reply:
It is required when executing as code because there isn't a terminal to answer yes. Could easily move this to `main.js` and check there.

Reply:
`main.js` sounds like a better place for this